File: src/com/rpsg/rpg/object/base/Item.java
package com.rpsg.rpg.object.base;

public abstract class Item {
	
	public boolean throwable=true;
	public String name;
	public int maxCount;
	public String illustration;
	public boolean disable=false;
	public abstract void use();
	
	public String toString(){
		return name;
	}
}

File: extension/RPSGItemEditor/ItemEditor/ItemEditor/Item.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ItemEditor
{
    class Item
    {
        public String name;
        public String realName;
        public int id;
        public String filePath;
        public String type;
        public bool throwable;
        public String illustration;
        public bool disable;
        public String use;

        public override string ToString()
        {
            return name;
        }
    }
}
